packages/cli-dxns/src/handlers/seed.ts

```diff
@@ -19,6 +20,15 @@ const DEFAULT_SCHEMA_NAME = 'schema';
 
 const SCHEMA_PATH = path.join(__dirname, '../dxos.proto');
 
+const TYPES = {
```

Comment:
can we derive types from schema instead of hardcoding?

Reply:
The problem would be to pick which message definitions from protobuf should be registred as types and which are not. It might be possible to specify that with some metadata. I had to remove extensions for now, as they were triggering bugs in our tooling, and weren't used at the moment

Reply:
```suggestion
// TODO(dmaretskyi): Define types as protobuf metadata instead of hardcoding them
const TYPES = {
```